test/token/token-recovery.test.ts

```diff
@@ -40,20 +40,6 @@ describe('Token - Recovery', () => {
         });
       });
 
-      describe('when new wallet is not authorized on the identity', () => {
```

Comment:
We need to test the case in which 2 tokens that are linked to the same IRS are doing the recovery as it is what the bug was.

Reply:
in old implementation it should have worked on the first token and then failed on the second because the Identity Registry storage was already modified by the first recovery.